nordicNRPlus/thingExists.ts

```diff
@@ -0,0 +1,13 @@
+import { type IoTClient, DescribeThingCommand } from '@aws-sdk/client-iot'
+
+export const thingExists = async (
+	iotClient: IoTClient,
+	thingName: string,
+): Promise<boolean> => {
```

Comment:
```suggestion
export const thingExists = (
	iotClient: IoTClient) => async (
	thingName: string,
): Promise<boolean> => {
```

Bind dependencies once.